recipe/meta.yaml

```diff
@@ -1,40 +1,38 @@
-{% set version = "2.3.1" %}
+{% set version = "2.3.2.post1" %}
 
 package:
   name: soupsieve
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/s/soupsieve/soupsieve-{{ version }}.tar.gz
-  sha256: b8d49b1cd4f037c7082a9683dfa1801aa2597fb11c3a1155b7a5b94829b4f1f9
+  sha256: fc53893b3da2c33de295667a0e19f078c14bf86544af307354de5fcf12a3f30d
 
 build:
   number: 0
-  skip: True  # [py<36]
-  noarch: python
+  skip: True  # [py<37]
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
-    - python
+    - hatchling >=0.21.1
     - pip
-    - setuptools >=42
-    - wheel
+    - python
   run:
-    - python >=3.6
+    - python
 
 test:
   imports:
     - soupsieve
   requires:
-    - python <3.10
+    - python
     - pip
     - beautifulsoup4
   commands:
     - pip check
```

Comment:
Is it make sense to add `run_test.py` with examples from the home site https://facelessuser.github.io/soupsieve/ to check if the package works correctly?